website/views/shared/profiles/stats/active-pet-and-mount.jade

```diff
@@ -0,0 +1,5 @@
+div(ng-show='user._id != profile._id')
+  h4
+  table.table.table-striped
+    +basicRow('activePet', '{{::profile.items.currentPet ? (isBasicAnimal(profile.items.currentPet, \'pet\') ? env.t(\'petName\', {potion: Content.hatchingPotions[petPotion].text(), egg: Content.eggs[petEgg].text()}) : env.t(Content.specialPets[profile.items.currentPet])) : env.t(\'noActivePet\')}}')
```

Comment:
I did that, but nothing happened.
On Wed, Jul 20, 2016 at 5:12 PM Kaitlin Hipkin <notifications@github.com>
wrote:

> In website/views/shared/profiles/stats/active-pet-and-mount.jade
> <https://github.com/HabitRPG/habitrpg/pull/7768#discussion_r71616259>:
>
> > @@ -0,0 +1,5 @@
> > +div(ng-show='user._id != profile._id')
> > +  h4
> > +  table.table.table-striped
> > +    +basicRow('activePet', '{{::profile.items.currentPet ? (isBasicAnimal(profile.items.currentPet, \'pet\') ? env.t(\'petName\', {potion: Content.hatchingPotions[petPotion].text(), egg: Content.eggs[petEgg].text()}) : env.t(Content.specialPets[profile.items.currentPet])) : env.t(\'noActivePet\')}}')
>
> Hmm weird that it wouldn't return anything. It seems like this could be
> done with a few if-statements to check pet-vs-mount and basic-vs-special
> and return the appropriate strings.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7768/files/8a2482e7d895b3058629834e63ddf550c9f05f29#r71616259>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_80n2lhvWxNyk4B3sj2BoGXrYUE7Aks5qXp05gaJpZM4JIjd_>
> .
>